project.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Please don't commit the json file next time

Reply:
ok!